src/ServicePulse.Host/vue/src/composables/serviceServiceControlUrls.ts

```diff
@@ -148,11 +148,7 @@ function getParams() {
 }
 
 function getParameter(params: Param[], key: string) {
-  if (params) {
-    return params.find((param) => {
-      return param.name === key;
-    });
-  }
-
-  return undefined;
+  return params.find((param) => {
```

Comment:
does this still need to be null aware? i.e. params?.find((param) => param.name === key)

Reply:
The assumption here is that params should either be empty array or not, but not null.
I also checked the usages of this and they seem to always pass an array.